### PR TITLE
feat: add applyChange to AddObjectChange class

### PR DESCRIPTION
**Pull request by Cooryd**
Adds `applyChange` to the `AddObjectChange`.

### CODE REVIEW

File: test/unit/model/history/add-object-change.test.ts [hunk 1]
```diff
@@ -51,12 +51,53 @@ describe('AddObjectChange', () => {
...
+      public value1: string = '';
+      public value2: string = '';
+      constructor(val1: string, val2: string) {
+        super();
+        this.value1 = val1;
+        this.value2 = val2;
+      }
+    }
+
+    it('can add object to an empty manuscript array value', () => {
```
Comment (ilya-zhuravel) [inline test/unit/model/history/add-object-change.test.ts:64]: does it matter if the array is empty or populated?

**Review by ilya-zhuravel — APPROVED** [review]

File: test/unit/model/history/add-object-change.test.ts (hunk 1 above)
Comment (Cooryd) [reply test/unit/model/history/add-object-change.test.ts:64]: Probably doesn't offer much now but I like to add tests to empty and populated array values just incase theres any `.length` logic added in future.
